frontend/LoginPage.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+
+function LoginPage() {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleLogin = async () => {
+        const response = await fetch('/login', {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify({ email, password })
+        });
+
+        const data = await response.json();
+        console.log(data);
+    };
```

Comment:
check if data is null
```suggestion
        const data = await response.json();
        if (data !== null) {
            console.log(data);
        }
    };
```